src/components/app-navigator/TopTabNavigator.tsx

```diff
@@ -1,60 +1,77 @@
+import { ComponentType, ReactNode } from 'react'
```

Comment:
I like the updates here. Does the component still render and fill the screen as it needs to?

Reply:
yeah i ran through its usage in profile page and explore page and it seemed to still render in the right way. let's keep tabs on this though, no pun intented!